rhino.yml

```diff
@@ -129,14 +153,17 @@ services:
 #    video-h264-encoder-amd64:

```

Comment:
The `opendlv-video-h264-encoder` service does only work when `CLUON_SHAREDMEMORY_POSIX=0` !